docs/source/user_guide/configuration_options.rst

```diff
@@ -0,0 +1,890 @@
+==============================
+Receptor Configuration Options
+==============================
+
+---------------------
+Receptor command line
+---------------------
+
+Command line arguments have the form: receptor [--<action> [<param>=<value> ...] ...]
+
+<action> are listed below.  When an action has parameters, they are listed in their own section.
+
+-------------
+Miscellaneous
+-------------
+
+.. list-table:: Miscellaneous
+    :header-rows: 1
+    :widths: auto
+
+    * - Action
+      - Description
+    * - --bash-completion
+      - Generate a completion script for the bash shell
+    * - --config <filename>
+      - Load additional config options from a YAML file
+    * - --control-service
+      - Run a control service
+    * - --help
+      - Show this help
+    * - --local-only
+      - Run a self-contained node with no backends
+    * - --log-level
+      - Set specific log level output
```

Comment:
```suggestion
      - Specifies the verbosity level for command output.
```

Reply:
Where do you find the possible levels? What is the default?

Reply:
What about "Specifies the verbosity level for logging output." ?

The log levels are below along with the default.  Do we need a reference?

Reply:
Yeah, that suggestion sounds better. You could say something like "See the available verbosity levels in the following section for more information." as a reference but I would avoid any sort of linking (if that is even possible here.)